### DISCUSSION TITLE
Store and modify a config file without triggering a deployment in Next.js backend?

### DISCUSSION THREAD

**Discussion by GorvGoyl** [vercel/community#7340] [Help]
### Summary

I'm considering using a config file (like .json or .ts) for storing some data (like feature flags) in my Next.js backend. Is there a way to modify this config file without causing a redeployment each time changes are made?

This would help me avoid the round trip from a key-value store.

### Steps to Reproduce

--

**Comment by pawlean** [comment]
Hi @GorvGoyl,

If I understood your question correctly, maybe you could try using environment variables? Vercel allows you to set environment variables in your project settings, and they will be available to your Next.js application without requiring a redeployment.

**Reply by GorvGoyl** [reply]
I think the deployment is needed after updating env. Let me know if that's not the case.

**Reply by pawlean** [reply]
Oh, you're right. Sorry about that! Just tested it myself 🙈 

Let me see if I can find alternative workarounds. It may also be worth asking specifically in the [Next.js community](https://nextjs.org/docs/community).

[a comment by an automated account is not shown]
